src/worker/actions.ts

```diff
@@ -59,7 +59,7 @@ export function createWorkerMethods(vitest: Vitest[]): BirpcMethods {
   const vitestEntries = Object.entries(vitestById)
 
   function getId(vitest: Vitest) {
-    return vitest.server.config.configFile || vitest.config.workspace || vitest.config.root
+    return vitest.config.workspace || vitest.server.config.configFile || vitest.config.root
```

Comment:
I was thinking about injecting ID somehow - we know it in the `worker.ts` file

Reply:
I think I know what you mean. Yeah, probably `getId(vitest)` can be cleaned up since we have `initVitest(meta)` is already using `meta.id`.

I'll see what I can do.

Reply:
I removed `getId` in favor of constructing `vitestById` ahead of time https://github.com/vitest-dev/vscode/pull/322/commits/91f30446fe73d3bcb99e9e5e23b1b1a856cf34b8